### ISSUE TITLE
RGB and HSV differ in assumptions regarding coordinate order

### ISSUE THREAD

**Issue by PeterDSteinberg** [ioam/holoviews#1346]
I have a notebook with the following code:
```
from holoviews.operation.datashader import aggregate
import holoviews as hv
import numpy as np
import dask.array as da
import dask.dataframe as dd
from dask.distributed import Client
import geoviews as gv
from elm.readers import *
client = Client()
band_specs = [BandSpec('name',
                       'B{}.TIF'.format(idx),
                       'B' + str(idx),
                       buf_xsize=1000,
                       buf_ysize=1000)
              for idx in range(1, 8)]

pix = load_array(TIF_DIR, band_specs=band_specs) #elm.readers


dset = hv.Dataset(pix)

for b in dset.data.band_order:
    band = getattr(dset.data, b)
    band.values = band.values.astype(np.float64) / 2 ** 16


hv.HSV(dset, kdims=['x', 'y'], vdims=['B3', 'B4', 'B5'])
```
The HSV plot causes the image (below - Chesapeake Bay) to have north - south reversed:
<img width="1060" alt="screen shot 2017-04-21 at 4 40 58 pm" src="https://cloud.githubusercontent.com/assets/1445602/25299212/76d625f4-26b1-11e7-84af-a802141aeecc.png">

When changing just the final line of the snippet above to:
```
hv.RGB(dset, kdims=['x', 'y'], vdims=['B3', 'B4', 'B5'])
```
then image south - north orientation makes sense:
<img width="1338" alt="screen shot 2017-04-21 at 4 41 11 pm" src="https://cloud.githubusercontent.com/assets/1445602/25299234/a248a626-26b1-11e7-9fd0-cf831c341f68.png">

**Comment by philippjfr** [comment]
Thanks, I can reproduce, I forgot to update the ``HSV.rgb`` property after changing how it accesses the data. Simple fix and good to get in before 1.7.

**Comment by jlstevens** [comment]
The PR fixing this issue has now been merged. Closing.
